includes/emails/tags.php

```diff
@@ -288,7 +288,6 @@ function edd_email_tag_download_list( $payment_id ) {
 		return '';
 	}
 
-	$payment_data  = $payment->get_meta();
```

Comment:
@cklosowski noting specifically that I removed this because I'm now using the order's payment key directly and there is no need to get the payment meta.